ui/components/app/snaps/snap-install-warning/snap-install-warning.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import SnapInstallWarning from './snap-install-warning';
+
+export default {
+  title: 'Components/App/snaps/SnapInstallWarning',
+  component: SnapInstallWarning,
+  argTypes: {
+    onCancel: {
+      action: 'onCancel',
+    },
+    onSubmit: {
+      action: 'onSubmit',
+    },
```

Comment:
Function type props can use the action control. Updated here

Reply:
okay, I'll keep this in mind while adding new stories 👍